airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/writer/CommonWriter.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.gcs.writer;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.IOException;
+
+public interface CommonWriter {
```

Comment:
The three writer interfaces (S3Writer, GcsWriter, CommonWriter) may be consolidated. It's fine to leave them as is. However, can you create an issue for the future to investigate whether they can be merged into one?

Reply:
Merge S3Writer, GcsWriter, CommonWriter interfaces #8894

I think we also should think about moving working with files from the S3 destination to a common sub-project. Right now S3 destination is a part of the connector-base build.